src/ui/public/visualize/visualization_editor.js

```diff
@@ -29,7 +29,7 @@ uiModules
 
         $scope.renderFunction = () => {
           if (!$scope.vis) return;
-          editor.render($scope.visData, $scope.searchSource, getUpdateStatus($scope), $scope.uiState);
+          editor.render($scope.visData, $scope.searchSource, getUpdateStatus($scope.vis, $scope), $scope.uiState);
```

Comment:
this is probably not correct. editors should as well define what status updates they are interested in (and not use the definition of the visualization).

also it would be really nice that we wouldn't recalculate the updateStatus for editor and vis but would only do it once.